api/src/main/java/io/opencensus/trace/export/SpanData.java

```diff
@@ -111,6 +149,7 @@ public static SpanData create(
         parentSpanId,
         hasRemoteParent,
         name,
+        kind,
```

Comment:
Should we checkNotNull() here (since the arg `Kind kind` of this method is not marked `Nullable`)?

Reply:
Made it @Nullable. Actually null is a valid value.

Reply:
Should SpanBuilder.setSpanKind also use `@Nullable`?